src/gui/components/navbar/navbar.jsx

```diff
@@ -26,7 +26,7 @@ Navbar.propTypes = {
 
 Navbar.defaultProps = {
   style: {},
-  className: '',
+  className: undefined,
```

Comment:
because in `glamor#merge`, falsy values are : `false`, `null`, `undefined` and `{}`.
`''` is not considered as falsy.